LinearRegression/LinearRegressionBackend/DataProvider/DataConverter.cs

```diff
@@ -27,7 +28,33 @@ public static Vector<double> GetVector(double[,] imgPixelsMatrix)
 
         public static Matrix<double> ProcessInputImages(DirectoryInfo directory, IImageConverter converter)
         {
-            throw new NotImplementedException();
+            Vector<double> pixelVector;
+            Vector<double> labelVector;
+            Matrix<double> matrixOfPixels = null;
+            Matrix<double> matrixOfLabels = null;
```

Comment:
Shape of the label matrix `Y` is (l, m), where

- l: number of label.
- m: number of files in the folder

For instance if we have 100 image contains 50 circle and 50 square image then `Y` shape will be (2,100). Because we have only 2 label the **circle** and the **square**.